web/src/narrow_state.ts

```diff
@@ -106,14 +112,17 @@ export function set_compose_defaults() {
 
     if (single.has("dm")) {
```

Comment:
Remove this check; it’s subsumed by `private_message_recipient !== undefined`.

Reply:
Ah yeah of course. Cool, removed